src/main/resources/META-INF/jpa-schema-export.sql

```diff
@@ -55,6 +55,12 @@
 
     drop table if exists LetterLearningEvent;
 
+    drop table if exists LetterSound;
+
+    drop table if exists LetterSound_Letter;
+
+    drop table if exists LetterSound_Sound;
+
     drop table if exists LetterSoundContributionEvent;
 
     drop table if exists LetterSoundCorrespondence;
```

Comment:
@SnehaHS65 This file looks out-of-date because it shouldn't contain `LetterSoundCorrespondence` anywhere. Try re-generating it by running the `mvn jetty:run` command. And then commit the changes to your branch.

> [!NOTE]
> Note that this file does not get re-generated with `mvn clean compile`. It will only get re-generated after running the webapp on localhost.

Reply:
Done @jo-elimu